### DISCUSSION TITLE
Ability to group and collapse conversations?

### DISCUSSION THREAD

**Discussion by ssbarnea** [convos-chat/convos#588] [Q&A]
I am a heavy irc user, joined into 40+ freenode channels related to open-source. I ended up looking at Convos because my current irc client (TheLounge) does not have the ability to group/hide/collapse channels, something really needed when you have so many.

As some of these channels are low traffic or low importance, I want to group them under a collapsed folder so they do not clutter the sidebar. The folder should indicate if any notification was raised from the channels it contain, so you could expand it and read them.

If you plan to add this to conves, please let me know.

**Comment by jhthorsen** [comment]
Would #520 be of any help?

**Reply by ssbarnea** [reply]
To be honest now that I see that proposal, I do like it, and is probably far easier to implement than grouping. 

I am not 100% how it would **feel** as I may have some popular channels already moved to the top (first ~5) and if they are quite active I may end-up with some confusing reordering. Maybe if move to top is implemented as move to configurable position (1 would act as top) could be used to avoid reordering the first N ones.

**Comment by jhthorsen** [comment]
This should be fixed by @mostlyvirtual in #840.
